applications/API/filters.py

```
@@ -0,0 +1,47 @@
+    """
+    Provides filtering options for Insult objects in the API.
+
+    This filter allows users to filter insults by their NSFW status and category.
+    """
+
+    class Meta:
+        model = Insult
+        fields = {
+            "nsfw": ["exact"],
+            "category": ["exact"],
+        }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)